src/Reveal.Sdk.Dom.Tests/Visualizations/VisualizationFixture.cs

```diff
@@ -0,0 +1,445 @@
+﻿using Xunit;
+using Moq;
+using Reveal.Sdk.Dom.Data;
+using Reveal.Sdk.Dom.Visualizations;
+using System.Collections.Generic;
+using Xunit.Sdk;
+using Newtonsoft.Json.Linq;
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+using Reveal.Sdk.Dom.Core.Utilities;
+using System;
+
+namespace Reveal.Sdk.Dom.Tests.Visualizations
+{
+    public class VisualizationFixture
+    {
+        [Fact]
+        public void Constructor_SetTitleAndChartType_WhenConstructed()
+        {
+            // Arrange
+            var dataSource = new DataSource();
+            var dataSourceItem = new DataSourceItem { DataSource = dataSource };
+            var testTitle = "testTitle";
+
+            // Act
+            var visualization = new Mock<Visualization>(testTitle, dataSourceItem) { CallBase = true }.Object;
+
+            // Assert
+            Assert.Equal(testTitle, visualization.Title);
```

Comment:
In the constructor of `Visualization`, there is a call to `InitializeDataDefinition`. so I think we need more Asserts for code inside this method.
For ex:
![Uploading image.png…]()